src/PrestaShopBundle/Form/Admin/Sell/Product/EventListener/FeatureValueListener.php

```diff
@@ -93,7 +83,7 @@ public function updateFeatureValuesOptions(FormEvent $event): void
 
         if (empty($data['feature_id'])) {
             // make sure feature_value_id form is disabled in case it was rendered after error of not selecting feature_id
-            $newFeatureValueForm = $this->formCloner->cloneForm($form->get('feature_value_id'), ['disabled' => true]);
```

Comment:
this seemed to cause troubles. It must have been "attr" => disabled" or else it would still be disabled and avoid validation or something like that (not sure, it was quite tricky to find this :smile: )